shell/platform/tizen/tizen_window.h

```diff
@@ -6,71 +6,31 @@
 #ifndef EMBEDDER_TIZEN_WINDOW_H_
 #define EMBEDDER_TIZEN_WINDOW_H_
 
+#include "flutter/shell/platform/tizen/tizen_view_base.h"
+
 #include <cstdint>
 #include <string>
 #include <vector>
 
-#include "flutter/shell/platform/tizen/tizen_input_method_context.h"
-
 namespace flutter {
 
 class FlutterTizenView;
 
-class TizenWindow {
+class TizenWindow : public TizenViewBase {
  public:
-  struct Geometry {
-    int32_t left = 0, top = 0, width = 0, height = 0;
-  };
-
+  TizenWindow() = default;
   virtual ~TizenWindow() = default;
 
-  // Sets the delegate used to communicate state changes from window to view
-  // such as key presses, mouse position updates etc.
-  void SetView(FlutterTizenView* view) { view_ = view; }
-
-  // Returns the geometry of the current window.
-  virtual Geometry GetWindowGeometry() = 0;
-
-  // Set the geometry of the current window.
-  virtual void SetWindowGeometry(Geometry geometry) = 0;
-
-  // Returns the geometry of the display screen.
-  virtual Geometry GetScreenGeometry() = 0;
-
-  // Returns a valid pointer the platform object that rendering can be bound to
-  // by rendering backend.
-  virtual void* GetRenderTarget() = 0;
-
-  virtual void* GetRenderTargetDisplay() = 0;
+  virtual void* GetWindowHandle();
```

Comment:
Question: What's the difference between `GetWindowId` and `GetWindowHandle`? The both return a pointer (handle) to the underlying window instance but I can't clearly get what they are exactly for from their names. Can we refine this part as well?

Reply:
I think that both return value is different.

| renderer  | Id | Handle |
| ------------- | ------------- | ------------- |
| Elementary | ecore window | elm window |
| wl2 | ecore wl2 window id | ecore wl2 window |

I changed `ecore_evas_window_get(ecore_evas_ecore_evas_get(evas_object_evas_get(win_)))` to `elm_win_window_id_get(elm_win_);`.
elm_win_window_id_get() has the same behavior if the argument is elm_win.

Reply:
Thanks for the clarification!